Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -91,32 +101,74 @@ void HiresTexture::Update()
 
   const std::string& game_id = SConfig::GetInstance().GetGameID();
   const std::string texture_directory = GetTextureDirectory(game_id);
-  std::vector<std::string> extensions{
-      ".png", ".bmp", ".tga", ".dds",
-      ".jpg"  // Why not? Could be useful for large photo-like textures
-  };
+  std::vector<std::string> extensions{".dds", ".png", ".bmp",  ".tga",  ".tgb",
+                                      ".jpg", ".jpe", ".jpeg", ".jfif", ".hdr"};
 
-  std::vector<std::string> filenames =
+  std::vector<std::string> Filepaths_Found =
```

Comment:
lowercase underscores, `filepaths_found`. This goes for the rest of the variables too.